src/rust/engine/process_execution/src/remote_cache.rs

```diff
@@ -164,7 +167,7 @@ impl CommandRunner {
         };
         Ok(Some(output_file))
       }
-      None => Ok(None),
+      Some(directory::Entry::Symlink(_)) => todo!(),
```

Comment:
Error here for same reason as directory variant errors. An output _file_ was expected and no file was found.